jdi-light-angular-tests/pom.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>jdi-light-angular-tests</artifactId>
+    <name>Tests: JDI Angular</name>
+    <version>1.2.14</version>
+    <groupId>com.epam.jdi</groupId>
+
+    <properties>
+        <driver>chrome</driver>
+        <aspectj.version>1.9.5</aspectj.version>
+        <java.version>1.8</java.version>
+        <allure.testng>2.13.2</allure.testng>
+        <allure.maven>2.10.0</allure.maven>
+        <jetty.version>9.4.12.RC2</jetty.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.qameta.allure</groupId>
+            <artifactId>allure-testng</artifactId>
+            <version>${allure.testng}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.epam.jdi</groupId>
+            <artifactId>jdi-light-angular</artifactId>
+            <version>1.2.14</version>
+        </dependency>
+
+        <dependency>
```

Comment:
remove mock from angular tests. Move mocked tests in a separate project

Reply:
Removed, I'll add them to a new module in the next PR